backend/covid_resource_matcher/urls.py

```diff
@@ -17,6 +17,7 @@
 from django.urls import path, include
 
 urlpatterns = [
+    path("accounts/", include("covid.urls")),
```

Comment:
Don't apply *all* covid URLs to the `accounts/` route. Presumably this is for the user deletion endpoint; call out that url specifically (or even consider moving it to a separate urlpatterns)

Reply:
@Lucretiel; I'm not savvy on much Django. What do you mean here? Would:
`path("accounts/delete/", include("covid.urls"))`
do the trick?

Reply:
er... `path("deleteuser/", include("covid.urls")),`

Reply:
You're on the right track, but it'd be the opposite specialization. The problem with the code as written is that, for example, `/accounts/update_supplies` is a valid route, as is `/covid/deleteuser`. Your proposed solutions have a similar problem: `/accounts/delete/update_supplies` would be a valid route.

The solution depends on the final paths we want to have. The simplest solution would be to get rid of the `/accounts/` prefix and have all routes go through `/covid/` (eg, `/covid/deleteuser`, etc). In that case, we'd simply get rid of this line, and the account management routs would get picked up by `path("covid/", include("covid.urls"))`, below.

Feel free to email me if this still isn't clear; I'm happy to set up a video chat and give you a crash course in some of this stuff.